src/modules/tools/temperaturecontrol/PT1000.cpp

```diff
@@ -57,8 +57,8 @@ float PT1000::adc_value_to_temperature(uint32_t adc_value)
         return infinityf();
 
     // polynomial approximation for PT1000, using 4.7kOhm and 1kOhm (PT1000) 3.3V.
-
-    float x = (adc_value) / ((float)max_adc_value);
+    
```

Comment:
This makes absolutely no difference at  all to the logic, however it is cleaner.